### PR TITLE
fix: terraform destroy data-type and helm --set flag to pass secrets and multiple values.

### PR DESCRIPTION
**Pull request by h1manshu98**
### what
Terraform
- data-type warning when setting `destroy: true or false` in caller file.
- updated all conditions where `inputs.destroy` was used inside `if` statement for new `boolean` data-type

Helm
- Updated `set-parameter` to pass secrets and multiple values
- Added variable to `uninstall` helmchart
- Added a message in `rollback` to show valid revision values.

### CODE REVIEW

File: .github/workflows/helm.yml [hunk 1]
```diff
@@ -36,28 +36,36 @@ on:
         required: true
         type: string
         description: 'Timeout for helm install step in seconds'
```
Comment (VishwajitNagulkar) [inline .github/workflows/helm.yml:38]: Add default value for timeout

File: .github/workflows/helm.yml [hunk 2]
```diff
@@ -36,28 +36,36 @@ on:
...
-        description: 'Overriding the default values'
       values-file-path:
-        required: true
+        required: false
         type: string
         description: 'Values file path from helm chart directory'
       history-max:
         required: true
         type: number
         description: 'number of revisions stored in the revision history.'
```
Comment (VishwajitNagulkar) [inline .github/workflows/helm.yml:46]: Add default values and make it required it as false

File: .github/workflows/helm.yml (hunk 1 above)
Comment (h1manshu98) [reply .github/workflows/helm.yml:38]: When `required:` is set to `true`, the `default:` value will never be used, as the user has to pass a value.

File: .github/workflows/helm.yml (hunk 2 above)
Comment (h1manshu98) [reply .github/workflows/helm.yml:46]: When `required:` is set to `true`, the `default:` value will never be used, as the user has to pass a value.

**Review by VishwajitNagulkar — APPROVED** [review]
LGTM

**Review by AshutoshAM2002 — APPROVED** [review]
LGTM

**Review by Tanveer143s — APPROVED** [review]
LGTM!

**Review by VishwajitNagulkar — APPROVED** [review]

**Review by AshutoshAM2002 — APPROVED** [review]

**Review by Tanveer143s — APPROVED** [review]
LGTM

**Pull request merged**